src/cli/Extract.cpp

```diff
@@ -82,6 +82,13 @@ int Extract::execute(QStringList arguments)
             return EXIT_FAILURE;
         }
 
+        if (fileKey.type() != FileKey::Hashed) {
+            errorTextStream << QObject::tr("WARNING: You are using a legacy key file format which may become\n"
```

Comment:
@phoerious you should use `qWarn`, as this is a warning message that will not abort the execution of the command. Also, you could remove the `WARNING`, and simplify the message by saying the legacy key file format is deprecated.

Reply:
qWarning() is a debug tool. This is not a debug warning. Also qWarnings aren't translatable, since t expects const char*, while tr() returns a QString.